moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -142,7 +147,7 @@ class MoveGroup::MoveGroupImpl
                                (node_handle_, move_group::PLACE_ACTION, false));
     waitForAction(place_action_client_, wait_for_server, move_group::PLACE_ACTION);
 
-    execute_service_ = node_handle_.serviceClient<moveit_msgs::ExecuteKnownTrajectory>(move_group::EXECUTE_SERVICE_NAME);
+    // execute_service_ = node_handle_.serviceClient<moveit_msgs::ExecuteKnownTrajectory>(move_group::EXECUTE_SERVICE_NAME);
```

Comment:
so this new ExecutePathAction replaced the old ExecuteKnownTrajectory service, correct? but we need a backwards compatible way of phasing out the old one - this won't work